src/app/src/App.js

```diff
@@ -9,22 +8,21 @@ import Footer from './Footer';
 import SensorOverview from './SensorOverview';
 
 import { POLLING_INTERVAL } from './constants';
-import SENSORS from './sensors';
-
-import { pollSensor } from './sensor.actions';
-
+import { pollSensor } from './app.actions';
+import { getSensorByProp } from './sensorUtils';
 
 class App extends Component {
     componentDidMount() {
         // Poll for new sensor data immediately and on a timed cycle thereafter
-        const {
-            dispatch,
-        } = this.props;
-        const sensors = SENSORS.features;
-        const fetchSensorData = (sensor) => dispatch(pollSensor(sensor));
+        const { dispatch } = this.props;
```

Comment:
Not sure it's better or more legible, but we've been using `mapDispatchToProps` lately which supplants needing to bring `dispatch` in as a prop here and lets you prep the functions with props data too: https://github.com/open-apparel-registry/open-apparel-registry/blob/develop/src/app/src/components/FacilityListItems.jsx#L167

One quirk is that if `this.props.sensors` is gotten via `mapStateToProps`, it may not be available as an argument in `mapDispatchToProps`. https://react-redux.js.org/using-react-redux/connect-mapdispatch#arguments describes using an `ownProps` function, but it `ownProps` may also include the `mapStateToProps` stuff since that function appears to be applied before `mapDispatchToProps`.

Reply:
Thanks. I looked at that example you provided earlier, and decided to go the other route because of it needing to use props itself. I wasn't sure how to sort that out. I'll take a look `ownProps`.

Reply:
Question about the OAR example. Where does `match` argument come from?

Reply:
Mentioned this in Slack but the `match` object is available in that OAR example via React Router. The whole component's wrapped in a

```
<Route
    component={FacilityListItems}
    path="lists/:listID"
/>
```

invocation, which when called makes the parameter in the place of `:listID` available in the `FacilityListItems` component's props as `match.params.listID`.

Reply:
Thanks for your help on this. I'm going to leave as-is because you can't access props that are mapped in `mapStateToProps` in `mapDispatchToProps`. However, there may be some changes to this from #64.